tools/local_network.py

```diff
@@ -0,0 +1,534 @@
+#!/usr/bin/python3
```

Comment:
I don't think it's desirable to copy this file here. It is available under https://github.com/mobilecoinfoundation/mobilecoin/tree/33af560693f1d1d6a10f77863cd617b50c10dcd5/tools/local-network 
Having a copy here would create unnecessary maintenance burden. Is it possible to use the one in the `mobilecoin` submodule?